torch/nn/_functions/thnn/sparse.py

```diff
@@ -120,35 +120,37 @@ def backward(ctx, grad_output):
 
 class EmbeddingBag(Function):
 
-    def __init__(self, max_norm, norm_type, scale_grad_by_freq, mode):
-        super(EmbeddingBag, self).__init__()
-        self.max_norm = max_norm
-        self.norm_type = norm_type
-        self.scale_grad_by_freq = scale_grad_by_freq
-        self._indices = None
-        assert mode is not None
+    @staticmethod
+    def _renorm(ctx, indices, weight, max_norm, norm_type):
+        # clone indices since LookupTable_renorm modifies it in-place
```

Comment:
so the old version didn't clone indices?

Reply:
I believe so:
https://github.com/pytorch/pytorch/blob/a0ac72e84e2f14c42d1bfdd695c411c0f101487c/torch/nn/_functions/thnn/sparse.py#L173-L176